.travis.yml

```diff
@@ -1,14 +1,15 @@
 language: go
 
 go:
-- 1.8
```

Comment:
OK. For future reference, we should be able to specify each version we want to test on on a separate line if we eventually want to test on multiple versions.